app/src/main/res/layout/fragment_friend_requests.xml

```diff
@@ -6,13 +6,32 @@
     android:layout_height="match_parent"
     tools:context=".ui.friends.FriendRequests.FriendRequestsFragment"
     android:orientation="vertical">
+    <LinearLayout
+        android:id="@+id/friendrequestheader"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:background="@drawable/ic_header"
+        android:orientation="vertical"
+        android:padding="20dp">
+
+        <TextView
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginBottom="10dp"
+            android:fontFamily="@font/readexpro_semibold"
+            android:text="@string/title_friends"
```

Comment:
Change this title from "Freundeliste" to "Anfragen". Have to create a new resource string for that.